runtime/compiler/trj9/compile/J9Compilation.hpp

```diff
@@ -164,8 +164,7 @@ class OMR_EXTENSIBLE Compilation : public OMR::CompilationConnector
 
    bool pendingPushLivenessDuringIlgen();
 
-   TR::list<TR_MethodValueProfileInfo*> &getMethodVPInfos() { return _methodVPInfoList; }
-   TR::list<TR_MethodValueProfileInfo*> &getMethodHWVPInfos() { return _methodHWVPInfoList; }
+   TR::list<TR_ExternalValueProfileInfo*> &getExternalVPInfos() { return _externalVPInfoList; }
```

Comment:
A discussion comment in reply to this comment on why HWVPInfos can go - I assume it came from hardware profiling work and I'd just like to understand how we can clean this up without breaking anything.

Reply:
The intention was to treat them both as external value info in a single list, rather than adding one for each source of external info. However, I am not able to find where the hardware profiler hooks in to populate the original collection.

If hardware profiling were to be added again, it would have to add `TR_ExternalValueProfileInfo` on demand (as it would have with the original list) and potentially add special handling to `TR_ValueProfileInfoManager`, just as the IProfiler current does.

Reply:
@dsouzai you might be interested in this part of the change - any comments?

Reply:
I don't think anything was using `getMethodHWVPInfos`. Even when we were experimenting with Value Profiling with HWProfiling, we used these APIs 

https://github.com/eclipse/openj9/blob/2a24b14d040312a0ec54075a171070558a71e3be/runtime/compiler/trj9/compile/J9Compilation.hpp#L217-L231